app/src/main/java/com/github/orkest/domain/DeezerApiIntegration.kt

```diff
@@ -100,11 +100,12 @@ class DeezerApiIntegration {
             if (responseCode == HttpURLConnection.HTTP_OK) {
                 val inputStream = connection.inputStream
                 val response = inputStream.bufferedReader().use { it.readText() }
+                val regex = Regex("\\D+") // Define a regular expression to match non-digit characters
+                val cleared = regex.replace(response, "") // clear the string to a non -json format
                 Log.d("HELLO", response.toString())
-                //TODO NEED TO STORE THE PLAYLIST ID in the database so we can update it
-                completableFuture.complete(true)
+                completableFuture.complete(cleared)
             } else {
-                // TODO  Handle the error case here...
+                Log.d("DEEZER_API_FAIL", "Problems fetching the user in the deezer database")
             }
         }.start()
 
```

Comment:
Maybe use a constant for the deezer base url : const val DEEZER_API_BASE_URL = "https://api.deezer.com"

For the handling of the error case you could do : completableFuture.completeExceptionally(Exception("Failed to create a new playlist")), for better code usability.

Reply:
1) will fix a following commit 
2) As I said i will try to have a cleaner way to handle this in the future, but it is not trivial